### PR TITLE
Remove highlighting feature flag

### PR DESCRIPTION
**Pull request by PiotrKozlowski**
for: https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/847

### CODE REVIEW

**Comment by TomWoodward** [comment]
@PiotrKozlowski you missed some stuff. run `yarn tsc`

File: src/app/content/components/Page/highlightManager.ts [hunk 1]
```diff
@@ -58,7 +57,7 @@ const onSelectHighlight = (
   services.setPendingHighlight(highlight);
 });
 
-const createHighlighter = (services: Omit<Services, 'highlighter'>) => {
+export const createHighlighter = (services: Omit<Services, 'highlighter'>) => {
```
Comment (TomWoodward) [inline src/app/content/components/Page/highlightManager.ts]: why are you exporting this?

**Review by TomWoodward — CHANGES_REQUESTED** [review]
passing to QA

File: src/app/content/components/Page/highlightManager.ts (hunk 1 above)
Comment (PiotrKozlowski) [reply src/app/content/components/Page/highlightManager.ts]: At first, I was trying to mock this, but then I realized it is not needed and forget to remove this.

**Review by TomWoodward — APPROVED** [review]

**Pull request merged**